web/forms.py

```
@@ -1442,6 +1443,24 @@ class Meta:
         ]
 
 
+class TeamGoalCompletionForm(forms.ModelForm):
+    class Meta:
+        model = TeamGoalMember
+        fields = ["completion_image", "completion_link", "completion_notes"]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)